### PR TITLE
Security fix

### PR DESCRIPTION
**Pull request by carakas**
## Type

- Critical bugfix

## Pull request description

Got an email with some security holes in fork.
This pr should close them

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 # Libraries
 /vendor/*
+!/vendor/.htaccess
```
Comment (tijsverkoyen) [inline .gitignore]: Isn't this covered by https://github.com/forkcms/forkcms/blob/master/.htaccess#L17

File: .gitignore [hunk 2]
```diff
@@ -17,6 +18,7 @@
 /app/logs/*
 /app/sessions/*
 /bin/*
+!/bin/.htaccess
```
Comment (tijsverkoyen) [inline .gitignore]: Wouldn't it be more logical to include this in the general .htaccess-file?

File: .htaccess [hunk 3]
```diff
@@ -7,6 +7,26 @@ DirectorySlash Off
 # file etags (used when comparing local cached file to server file)
 FileETag MTime Size
 
+########## Begin - Common hacking tools and bandwidth hoggers block
+## By SigSiu.net and @nikosdion.
+# This line also disables Akeeba Remote Control 2.5 and earlier
+SetEnvIf user-agent "Indy Library" stayout=1
+# WARNING: Disabling wget will also block the most common method for
+# running CRON jobs. Remove if you have issues with CRON jobs.
+SetEnvIf user-agent "Wget" stayout=1
```
Comment (tijsverkoyen) [inline .htaccess]: I wouldn't include this by default

File: .gitignore (hunk 1 above)
Comment (carakas) [inline .gitignore]: true, I had missed that one

File: .htaccess (hunk 3 above)
Comment (carakas) [inline .htaccess]: fixed in 814f9bfa7

File: .gitignore (hunk 2 above)
Comment (carakas) [inline .gitignore]: fixed in 581e61c27

File: .gitignore (hunk 1 above)
Comment (carakas) [inline .gitignore]: fixed in 581e61c27

**Pull request merged**